synapse/handlers/space_summary.py

```diff
@@ -191,6 +280,11 @@ async def _get_child_events(self, room_id: str) -> Iterable[EventBase]:
         return (e for e in events if e.content.get("via"))
 
 
+@attr.s(frozen=True, slots=True)
+class _RoomQueueEntry:
+    room_id = attr.ib(type=str)
```

Comment:
Will the via stuff be added in a separate PR? It seems premature to separate this out.

Reply:
sorry, yes.